stf/tasks/utils/predictionjobloop.py

```diff
@@ -12,9 +12,9 @@ def __init__(
             context,
             stop_on_exception=False,
             random_order=True,
-            on_exception=None,
-            on_successful=None,
-            on_end=None,
+            on_exception_cb=None,
```

Comment:
what does "_cb" mean?

Reply:
cb is an abbreviation of callback, Should I use `on_exception_callback`  instead to make it even more clear?

Reply:
ah, yes `on_exception_callback` is more clear.

Reply:
8cd32c13b5672dcc77497f0c54968bdad0ce8b0f fixed.